### DISCUSSION TITLE
Remove the green cross ?

### DISCUSSION THREAD

**Discussion by Mitia99** [flxzt/rnote#827] [Q&A]
Hi,

Is there a way to remove the green cross?

![image](https://github.com/flxzt/rnote/assets/56766778/e0ffbdd5-bcf6-4b12-b526-124d170cae8c)

Thanks,

**Comment by flxzt** [comment]
Currently not, no. What would be your use case that requires/would be improved by hiding it?

**Reply by Mitia99** [reply]
There is no specific case but I mostly use rnote as a Whiteboard so I'd like to have the layout as clean as possible. I already disabled patterns but I have to move the canva in order to hide the green cross. It's not that convenient.
#830

**Comment by flxzt** [comment]
closing this for #830
